### PR TITLE
Fixed imports, whitespaces and other pep8 errors

### PR DESCRIPTION
**Pull request by peterdemin**

### CODE REVIEW

**Comment by peterdemin** [comment]
Changes could look destructive, but most of them were applied automatically by autopep8.
I separated these cosmetic changes from actual bug fix in following PR.

File: src/RequestsLibrary/RequestsKeywords.py [hunk 1]
```diff
@@ -60,31 +57,34 @@ def f_retry(self, *args, **kwargs):
...
         self.mbackoff = 0.0
-        
+
     def _utf8_urlencode(self, data):
-        if type(data) is unicode:
+        if isinstance(data, unicode):
             return data.encode('utf-8')
 
-        if not type(data) is dict:
+        if not isinstance(data, dict):
```
Comment (bulkan) [inline src/RequestsLibrary/RequestsKeywords.py]: is this a change by the pep8 tool ?

Comment (peterdemin) [inline src/RequestsLibrary/RequestsKeywords.py]: Autopep8 to be precise

**Comment by vkosuri** [comment]
@peterdemin thank for your support, @oleduc already made many changes in PR https://github.com/bulkan/robotframework-requests/pull/86 if so could you please close this PR?

**Pull request closed without merging**